.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Submit a feature request or bug report
+
+Replace any `X` with your information.
+
+- [ ] This issue request isn't a duplicate of an [existing one](https://github.com/roots/trellis/issues)
+- [ ] I've read the [docs](https://roots.io/trellis/docs) and followed them (if applicable)
+
+---
+
+### Is this a request for a feature or a bug report?
+
+- [ ] Feature request
+- [ ] Bug report
+
+### What is the current behavior?
+
+X
+
+### If this is a bug report, please provide steps to reproduce and log output:
+
+X
+
+### What is the expected or desired behavior?
+
+X
+
+### Please provide use cases for changing the current behavior (for a feature request):
+
+X
+
+### Please describe your development environment (OS, Ansible Version, Vagrant version, etc):
+
+X
```

Comment:
I would actually list out the things we ask for here like 

Ansible version: X
OS: X
Vagrant Version: X